apps/bleprph/src/main.c

```diff
@@ -292,9 +292,6 @@ main(void)
 {
     int rc;
 
-    /* Set initial BLE device address. */
-    memcpy(g_dev_addr, (uint8_t[6]){0x0a, 0x0a, 0x0a, 0x0a, 0x0a, 0x0a}, 6);
```

Comment:
just use static random so that it doesn't use all zero address in combined builds on nRF5x